### PR TITLE
Rephrase the explanation of using shared contexts via metadata

### PR DESCRIPTION
**Pull request by mxie**
* Break up a long sentence.
* Fix "context" misspelling.

### CODE REVIEW

File: features/example_groups/shared_context.feature [hunk 1]
```diff
@@ -4,7 +4,12 @@ Feature: shared context
   of example groups either explicitly, using `include_context`, or implicitly by
   matching metadata.
 
-  When implicitly including shared contexts via matching metadata, the normal way is to define matching metadata on an example group (in which case the ontext is included in the entire group), but you can also include it in an individual example. RSpec treats every example as having a singleton example group (analogous to Ruby's singleton classes) containing just the one example.
+  When implicitly including shared contexts via matching metadata, the normal
+  way is to define matching metadata on an example group, in which case the
+  context is included for the entire group. However, you also have the option to
+  include it in an individual example instead. RSpec treats every example as
+  having a singleton example group (analogous to Ruby's singleton classes)
+  containing just the one example.
```
Comment (myronmarston) [inline features/example_groups/shared_context.feature]: When this renders on relishapp, it renders a line break at every line break in the markdown (which is different from most markdown renderers).  As a result, we are migrating to making each paragraph a single, long line, which then will wrap on relish appropriately.

Please leave this as a single, long line.

Comment (mxie) [inline features/example_groups/shared_context.feature]: Not a problem! Fixed in b0bf462.

File: features/example_groups/shared_context.feature [hunk 2]
```diff
@@ -4,7 +4,7 @@ Feature: shared context
   of example groups either explicitly, using `include_context`, or implicitly by
   matching metadata.
 
-  When implicitly including shared contexts via matching metadata, the normal way is to define matching metadata on an example group (in which case the ontext is included in the entire group), but you can also include it in an individual example. RSpec treats every example as having a singleton example group (analogous to Ruby's singleton classes) containing just the one example.
+  When implicitly including shared contexts via matching metadata, the normal way is to define matching metadata on an example group, in which case the context is included for the entire group. However, you also have the option to include it in an individual example instead. RSpec treats every example as having a singleton example group (analogous to Ruby's singleton classes) containing just the one example.
```
Comment (myronmarston) [inline features/example_groups/shared_context.feature]: Thanks...it's much easier to read this diff this way, too :).

I think that "in" reads better than "for" here

> in which case the context is included in the entire group

vs.

> in which case the context is included for the entire group

Comment (mxie) [inline features/example_groups/shared_context.feature]: Done. :)

**Comment by myronmarston** [comment]
Thanks!

**Comment by mxie** [comment]
Awesome! Thank you so much for taking a look at this!

**Pull request merged**